src/components/Article.jsx

```diff
@@ -0,0 +1,92 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { fetchArticleById, fetchCommentsByArticleId } from "../api";
+import {
+	Card,
+	Container,
+	Spinner,
+	Button,
+	DropdownButton,
+} from "react-bootstrap";
+import { capitaliseFirstLetter } from "../utils/utils";
+
+function Article() {
+	const { article_id } = useParams();
+
+	const [article, setArticle] = useState(false);
+	const [votes, setVotes] = useState(0);
+	const [comments, setComments] = useState([]);
+
```

Comment:
would make sense to introduce loading state at this point, it's really useful throughout the app when i throttle my internet to know the app is still working just taking some time.

Reply:
Loading is already implemented. It is just dependent on the article state, rather than an isLoading state.